app/__init__.py

```diff
@@ -9,14 +9,14 @@
 migrate = Migrate()
 load_dotenv()
 
+token = os.environ.get("API_TOKEN")
```

Comment:
This token isn't relevant to this project start up code. Prefer that you access this value in the same place that you make the request to Slack (helpers.py), instead of setting `token` here and importing it elsewhere.